package.json

```diff
@@ -35,6 +35,7 @@
     "@material-ui/lab": "~4.0.0-alpha.42",
     "@material-ui/pickers": "~3.2.10",
     "@material-ui/styles": "~4.9.0",
+    "babel-plugin-transform-imports": "^2.0.0",
```

Comment:
Without looking into it further, this seems like it's only a `devDependency`. Please remove.